### PR TITLE
PS 8347 add check for warnings ps 80 test

### PR DESCRIPTION
**Pull request by panchal-yash**
Add warnings check for ps 80  using the env check_warnings

### CODE REVIEW

File: playbooks/ps_80.yml [hunk 1]
```diff
@@ -101,6 +101,13 @@
   - name: install percona-mysql-shell package
     include_tasks: ../tasks/install_pshell.yml
 
+  - name: check that Percona Server version is correct
```
Comment (eleo007) [inline playbooks/ps_80.yml:104]: Why do we check version for 2 times? (the same is on line 111).

File: playbooks/ps_80.yml [hunk 2]
```diff
@@ -101,6 +101,13 @@
...
+    shell: /package-testing/version_check.sh ps80 &> version-check.log
+
+  - name: Log check for warnings in the previos stage
+    command: /package-testing/scripts/log-warning-check.sh version-check.log
+    when: (lookup('env', 'check_warnings') == "yes")
```
Comment (eleo007) [inline playbooks/ps_80.yml:109]: Please provide link to working job where the warning is detected.

**Comment by panchal-yash** [comment]
Link to working job where the warning is detected.
Link of the console output: https://ps80.cd.percona.com/job/package-testing-ps80-build-yash-test/33/console
Link of the parameter values used: https://ps80.cd.percona.com/job/package-testing-ps80-build-yash-test/33/parameters/

**Comment by eleo007** [comment]
> Link to working job where the warning is detected. Link of the console output: https://ps80.cd.percona.com/job/package-testing-ps80-build-yash-test/33/console Link of the parameter values used: https://ps80.cd.percona.com/job/package-testing-ps80-build-yash-test/33/parameters/

The provided links are for client check. Please provide links for ps80.

**Comment by panchal-yash** [comment]
Links to the jenkins job where warning is detected for ps80
Link for the console output: https://ps80.cd.percona.com/job/package-testing-ps80-build-yash-test/72/console
Link for the parameter values used: https://ps80.cd.percona.com/job/package-testing-ps80-build-yash-test/72/parameters/

**Pull request merged**